spec/models/reporting/patient_states_per_month_spec.rb

```diff
@@ -0,0 +1,354 @@
+require "rails_helper"
+
+RSpec.describe Reporting::PatientStatesPerMonth, {type: :model, reporting_spec: true} do
+  describe "Associations" do
+    it { should belong_to(:patient) }
+  end
+
+  def ltfu_patient_ids(month_date: Date.current.beginning_of_month)
+    described_class
+      .where(htn_care_state: "lost_to_follow_up", month_date: month_date)
+      .pluck(:id)
+  end
+
+  def under_care_patient_ids(month_date: Date.current.beginning_of_month)
```

Comment:
Thanks Rob. I Aaree with the helpers obscuring it, I've inlined it with 9067d25c. I want to make the enum change as well because we'll likely use the values in calling locations, will do that in the following PR.

I'm wary of adding scopes at this time, however. I would like to keep the reporting query interface minimal, and not provide too many options to start with.

Reply:
I also think the specs are not the right drivers for this decision. Let's start using this model for actual reports first. Once we get a feel for what that looks like, let's let that drive the enum decision for us.